Travel Scheduler/Test/Schedule.m

```diff
@@ -7,70 +7,269 @@
 //
 
 #import "Schedule.h"
+#import "TravelSchedulerHelper.h"
+#import "APIManager.h"
+#import "Place.h"
 #import "Date.h"
+#import "NSArray+Map.h"
+
+#pragma mark - Algorithm helper methods
+
+static int getNextTimeBlock(TimeBlock timeBlock) {
+    if (timeBlock == TimeBlockEvening) {
+        return 0;
+    }
+    return timeBlock + 1;
+}
+
+static int getDayOfWeekAsInt(NSDate *date) {
+    NSString *dayString = getDayOfWeek(date);
+    if ([dayString isEqualToString:@"Monday"]) {
+        return 1;
+    } else if ([dayString isEqualToString:@"Tuesday"]) {
+        return 2;
+    } else if ([dayString isEqualToString:@"Wednesday"]) {
+        return 3;
+    } else if ([dayString isEqualToString:@"Thursday"]) {
+        return 4;
+    } else if ([dayString isEqualToString:@"Friday"]) {
+        return 5;
+    } else if ([dayString isEqualToString:@"Saturday"]) {
+        return 6;
+    } else if ([dayString isEqualToString:@"Sunday"]) {
+        return 0;
+    }
```

Comment:
I think it is a good idea to use enums here

Reply:
Will do.